copy.go

```diff
@@ -6,7 +6,16 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sync"
+	"sync/atomic"
 	"time"
+
+	"github.com/jaypipes/ghw"
+)
+
+var (
+	_hasAllSSDsCalled atomic.Bool
```

Comment:
`atomic.Bool` doesn't exist on Go1.18. Let me know your thought for backward compatibility.
If we can discuss and make a decision, let us leave Go1.18 out of support. (I guess we wouldn't do that)

Reply:
Good catch! If we move the call to `hasAllSSDs()` into the `Copy()` function, I don't think we need these variables any more.

Reply:
Same as https://github.com/otiai10/copy/pull/120#discussion_r1309416320.